tests/cli/commands/local_commands/test_api_server_command.py

```diff
@@ -63,7 +63,7 @@ class TestCliFastAPI(_CommonCLIGunicornTestClass):
             ),
         ],
     )
-    def test_cli_fastapi_api_debug(self, app, args, expected_command):
+    def test_cli_fastapi_api_dev(self, app, args, expected_command):
```

Comment:
Should `fastapi_api` still appear in test names ?

Reply:
Good catch, I just missed it.

Reply:
Updated them. Give it a look.